components/List.tsx

```diff
@@ -1,19 +1,115 @@
-import * as React from 'react'
+import React, { useEffect } from 'react'
```

Comment:
Since this component is a list of all the songs in a playlist can you please name this file Playlist.tsx? And name the List Item as Track.tsx for now - I think in the end we won't even need a separate component for each track since the user won't see it.